.github/setup-node/action.yml

```diff
@@ -0,0 +1,58 @@
+name: 'Setup Node'
```

Comment:
I'm not entirely happy with the name. Maybe we should make it explicit that it also handles installing dependencies?

Reply:
Hmm, how about something like "Configure Node.js and install npm dependencies while managing all aspects of caching"?

It looks like this `name` is not displayed anywhere. For example, in [this workflow associated with this PR](https://github.com/WordPress/gutenberg/actions/runs/4111626373/jobs/7095591254#step:3:1), the text displayed for the step calling this composite action is the one defined in that respective workflow. I'm not sure where this will display, actually. May be better to give a full short description here.

Reply:
I changed the name a bit and moved your suggestion to the `description` field.